### ISSUE TITLE
I had the pleasure of reading your paper,but there was a problem that...

### ISSUE THREAD

**Issue by ChuxiaYang** [chenhao2345/ABMT#8]
Do you have any other tricks in your code? Why can't I get your secore with GPU 1*3060 and 1*V100?

**Comment by chenhao2345** [comment]
According to my experience, the combination of gpu number, batchsize and learning rate can always affect results. 

I used 4 GPUs to train. Thus, the batchsize and learning rate values in our paper are set based on 4 GPUs. 

If you have enough GPUs, try to use 4 GPUs. Otherwise, you may need to search optimal batchsize and learning rate values to find best results with 2 GPUs. Wish this can help you.

**Comment by ChuxiaYang** [comment]
Thanks for your reply.Let me try again.
